hack/BUILD.bazel

```diff
@@ -273,3 +277,8 @@ filegroup(
     tags = ["automanaged"],
     visibility = ["//visibility:public"],
 )
+
+sh_library(
+    name = "utils",
+    srcs = ["utils.sh"],
+)
```

Comment:
Something to do with `update-codegen.sh` 

```
git grep utils.sh
hack/BUILD.bazel:    srcs = ["utils.sh"],
hack/update-codegen.sh:source "$(rlocation com_github_jetstack_cert_manager/hack/utils.sh)"

```

Reply:
It is added as a dep in the `update-codegen` target [here](https://github.com/irbekrm/cert-manager/blob/bump_go_and_deps/hack/BUILD.bazel#L168). It isn't actually needed for the functionality, but I thought that we need to make Bazel aware of it being a dependency so that it knows to re-run the `update-codegen` if the utils script changes.